libbeat/tests/compose/wrapper.go

```diff
@@ -195,6 +195,22 @@ func (d *wrapperDriver) Up(ctx context.Context, opts UpOptions, service string)
 		return errors.Wrapf(err, "failed to pull images using docker-compose: %s", stderr.String())
 	}
 
+	// Try to build the docker images
+	build := d.cmd(ctx, "build", "--pull", service)
+	build.Stdout = nil
+	build.Stderr = &stderr
+	if err := build.Run(); err != nil {
+		fmt.Println(">> Building docker images again (after 10 seconds)")
+		// This sleep is to avoid hitting the docker build issues when resources are not available.
+		time.Sleep(10)
+		build := d.cmd(ctx, "build", service)
+		build.Stdout = nil
+		build.Stderr = &stderr
+		if err := build.Run(); err != nil {
+			fmt.Printf(">> Building docker images failed using docker-compose (let's try now with docker-compose up --build): %s", stderr.String())
+		}
+	}
+
```

Comment:
There are already some retries in the higher-level [`EnsureUp` method](https://github.com/elastic/beats/blob/addadf2f692949a3b90c2825adb150acf952c7d0/libbeat/tests/compose/compose.go#L99), that is the one used out of this package.

Also, we shouldn't add a explicit build here, we should take into account `opts.Create.Build` and so on.

I think this code is not needed.